provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerController.java

```diff
@@ -95,8 +75,45 @@ public PractitionerController(
   }
 
   @SneakyThrows
-  private ProviderContacts ppmsProviderContact(String id) {
-    return client.providerContactsForId(id);
+  private ProviderContactsResponse providerContact(String id) {
+    return ppmsClient.providerContactsForId(id);
```

Comment:
I think this one-line method is a relic from before the `PpmsClient` existed, when there was a lot more logic here. Now it's not really buying anything, so let's remove this and just invoke `ppmsClient` directly where this method is used.